### ISSUE TITLE
Question about IdleStrategy

### ISSUE THREAD

**Issue by guidomedina** [real-logic/agrona#155]
Hi,

I'm going to be writing a wheel spinner next week which will scan non-blocking queues, say I have 16 queues and 4 threads, the 16 queues are used to put elements on a `queue[x]` where `x = Math.abs(hash of message % 16)`

Is this a proper way to use the an `IdleStrategy`?
```
while(running) {
  // might stop before the last queue because threads are all in used and next time will continue where it was left off.
  // It will return true if after a full scan no task was rejected.
  boolean hasMoreWorkToDo = scanQueuesAndSubmitTasks();
  if (!hasMoreWorkToDo) {
    idleStrategy.idle();
  }
}
```

I saw the JavaDoc of `IdleStrategy` but still have some doubts.

**Comment by mjpt777** [comment]
Yes this will work for most strategies. To work well with a backoff strategy then call `idleStrategy.reset()` when work was done.

**Comment by mjpt777** [comment]
Or more simply return a work count and pass that to `idleStrategy.idle(workCount)`.

**Comment by guidomedina** [comment]
Ok, I think I got it, I looked at the JavaDoc and completely missed `idle(int workCount)` method, so I can simple assign 1 to each queue check and that's it, this is perfect, makes sense now, thanks!
